### DISCUSSION TITLE
Works perfectly with Rio spa and square 2021 pump

### DISCUSSION THREAD

**Discussion by VincentCorniglion** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#303] [Show and tell]
Hello,

I wanted to thank you very much for this project. It's working perfectly with my layzspa Rio with a square 2021 pump ref S100101.
For information I didn't face any of the issues raised in other chats (e.g. blinking screen).
I used TXS0108E level converter and didn't use any resistance: direct links between ESP and the level converter and same between level converter and DSP/Controller.
So far I have a working prototype with a breadboard PCB, but I plan to order a proper PCB now that I know it's working!

Now my spa is remotely managed thanks to home assistant! Finally able to automate the filtering pump scheduling and allowing me to startup heater remotely!

Again thanks a lot!

**Comment by visualapproach** [comment]
Awesome! Not overly exposed yet but feel free to use pcb_v2. I think it's better. Maybe only found in dev branch yet.

**Comment by Remigiusz2020** [comment]
> Hello,
> 
> I wanted to thank you very much for this project. It's working perfectly with my layzspa Rio with a square 2021 pump ref S100101. For information I didn't face any of the issues raised in other chats (e.g. blinking screen). I used TXS0108E level converter and didn't use any resistance: direct links between ESP and the level converter and same between level converter and DSP/Controller. So far I have a working prototype with a breadboard PCB, but I plan to order a proper PCB now that I know it's working!
> 
> Now my spa is remotely managed thanks to home assistant! Finally able to automate the filtering pump scheduling and allowing me to startup heater remotely!
> 
> Again thanks a lot!

hi, I have also pump s100101 but model Fiji. I have just started with this project. Can you write me more detalis..... which PCB, llc end converter ? resistor end others thing ? many thanks
